third-party/cmake/BuildLuarocks.cmake

```diff
@@ -1,35 +1,89 @@
-option(USE_BUNDLED_BUSTED "Use the bundled version of busted to run tests." ON)
+# Luarocks recipe. Luarocks is only required when building Neovim, when
+# cross compiling we still want to build for the HOST system, whenever
+# writing a recipe than is mean for cross-compile, use the HOSTDEPS_* variables
+# instead of DEPS_* - check the main CMakeLists.txt for a list.
 
-if(USE_BUNDLED_LUAJIT)
-  list(APPEND LUAROCKS_OPTS
-    --with-lua=${DEPS_INSTALL_DIR}
-    --with-lua-include=${DEPS_INSTALL_DIR}/include/luajit-2.0)
+if(MSVC)
+  message(STATUS "Building busted in Windows is not supported (skipping)")
```

Comment:
It should be undefined anyway since the option(...) is never called - I don't think its bad if someone wants to explicitly force -DUSE_BUNDLED_BUSTED=ON to try it anyway. The problem with busted is not in our recipe, its upstream in nvim-client and lua-xml.